src/astro/sql/table.py

```diff
@@ -87,7 +95,7 @@ def create_table_name(context) -> str:
     dag_run: DagRun = ti.get_dagrun()
     table_name = f"{dag_run.dag_id}_{ti.task_id}_{dag_run.id}".replace(
         "-", "_"
-    ).replace(".", "__")
+    ).replace(".", "__")[:MAX_TABLE_NAME_SIZE]
```

Comment:
This approach can have side effects, particularly if the first 63 characters of the table name are the same across different tasks. @dimberman @utkarsharma2 what do you think would be a good strategy?

We probably should improve this as part of a separate ticket.

Reply:
@tatiana hard to say, I believe that's a restriction that is given to us from elsewhere (e.g. certain dbs have table name size limits)

Reply:
@dimberman https://github.com/astro-projects/astro/issues/218